### ISSUE TITLE
SCP:SL Exiled Console Command not working

### ISSUE THREAD

**Issue by LEON-BROTHER** [parkervcp/eggs#2743]
### Panel Version

1.11.8

### Wings Version

1.11.8

### Service

gameserver/steamcmd/scpsl/exiled

### Modified

No, I did not modify the egg

### Expected Behavior

That the command will execute

### Actual Behavior

It does show it in the console but it’s not executing the command

### Steps To Reproduce

Import the egg to a nest
Start the server
Wait
Enter HELLO

### Install logs

Im not on my pc rn sorry

**Comment by QuintenQVD0** [comment]
Did it ever work? As many games do not have a working input console. Or never had a working one in pterodactyl

**Comment by LEON-BROTHER** [comment]
I don't think it ever worked but it should work tho if you run it outside of pterodactyl

**Comment by gOOvER** [comment]
console works fine for Setup, but Not after Setup, because Ptero have Not a real console.

use the commands in Game.

Not really an Egg issue

**Comment by LEON-BROTHER** [comment]
okay but is there any alternative to access the real console cus i kinda need it
